main.go

```diff
@@ -257,6 +260,10 @@ func main() {
 	flag.StringVar(&tableName, "table", "test", "table to use")
 
 	flag.StringVar(&hostSelectionPolicy, "host-selection-policy", "token-aware", "set the driver host selection policy (round-robin,token-aware,dc-aware),default 'token-aware'")
+
+	flag.StringVar(&cqlUsername, "cql-username", "", "cql username for authentication")
```

Comment:
Why would you want it to be prefixed with `cql`, how about just username and password - applies to both variables and flags.

Reply:
ok, I can leave the cql word in the variable description only